src/tokens/solana.tokenlist.json

```diff
@@ -65973,6 +65973,17 @@
       "tags": [
         "social-token"
       ]
+    },
+    {
+      "chainId": 101,
+      "address": "swimnKEr963p7EbCjsSnBCoYwytuZHPm3zbq6fKLHXb",
+      "symbol": "SWIM",
+      "name": "SWIM Protocol Token",
```

Comment:
Not sure about this, maybe we just want `SWIM` here too?

Reply:
I asked about this and because the other SWIM token (the one related to the sol-aquariums NFT project) has the value of "SWIM" for the name it would be better to be more explicit here. It seems like this will be the value for how the names will show up in phantom wallet so just one additional way to avoid any type of confusion with the other token (and any other potential collisions in the future) especially if other protocols/dApps will be programmatically pulling the token list and just rendering/displaying the data as is.